src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidDescriptionException;
+
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * Ui class handles the input
+ *
+ * @author Justin Ng Jie Ern
+ */
+public class Ui {
+
+    /**
+     * Constant String from Duke after every command to Duke.
+     */
+    private final static String FROM_DUKE = "From Duke: \n\t";
+
+    /**
+     * Storage Class to help with loading or saving or file.
+     */
+    Storage storage;
+
+    /**
+     * TaskList Object is to help loaded and save.
+     */
+    TaskList taskList;
+
+    /**
+     * Constructor to create a Ui Object.
+     *
+     * @param taskList Constructor to create a Ui Object.
+     * @param storage Storage Class to help with loading or saving or file.
+     */
+    public Ui(TaskList taskList, Storage storage) {
+        this.storage = storage;
+        this.taskList = taskList;
+    }
+
+    /**
+     * Method to run Duke.
+     */
+    public void uiHandler() {
```

Comment:
Would using a verb for your method be better?

Reply:
Changed to run() instead. Thanks